Header header
float32[] xmin
float32[] ymin
float32[] xmax
float32[] ymax
float32[] confidence
uint32[] object_class
float32[] x
float32[] y
float32[] w
float32[] h